README.md

```diff
@@ -395,21 +395,21 @@ interface State {
 }
 
 // Will only invalidate when the foo property is changed
-const Container = Container(WidgetBase, 'state', { paths: [ [ 'foo' ] ], getProperties(store: Store<State>) {
+const Container = StoreContainer<State>(WidgetBase, 'state', { paths: [ [ 'foo' ] ], getProperties(store: Store<State>) {
 	return {
 		foo: store.get(store.path('foo'))
 	}
 }});
 
 // Catch all, will invalidate if _any_ state changes in the store even if the container is not interested in the changes
-const Container = Container(WidgetBase, 'state', { getProperties(store: Store<State>) {
+const Container = StoreContainer<State>(WidgetBase, 'state', { getProperties(store: Store<State>) {
 	return {
 		foo: store.get(store.path('foo'))
 	}
 }});
 ```
 
-To provide a typed container for the store's state, use the `createStoreContainer` function from `@dojo/stores/StoreInjector` passing in the state interface as a generic and then export the returned `Container` to be available throughout your application.
+To provide a create typed container for the store's state, use the `createStoreContainer` function from `@dojo/stores/StoreInjector` passing in the state interface as a generic and then export the returned `Container` to be available throughout your application.
```

Comment:
"To provide a create typed container" while this might be accurate, this is difficult to follow.

Reply:
How about:

----

Instead of typing `StoreContainer` for every usage, it is possible to create a pre-typed StoreContainer that can be used across your application. To do this use `createStoreContainer` from `@dojo/stores/StoreInjector` passing the stores state interface as the generic argument. The result of this can be exported and used across your application.

```ts
export const StoreContainer = createStoreContainer<StateInterface>();
```